### PR TITLE
Fix the next composition of combined consonant in Korean

### PR DESCRIPTION
**Pull request by mshin-wolvic**
This CL fixes the next composition issue of combined consonant when vowel is added in Korean.

Example : `않` + `ㅏ` should be `안하` instead of `않ㅏ`.

### CODE REVIEW

**Review by jdapena — APPROVED** [review]

**Review by duswnchl — COMMENTED** [review]
What do you think this PR be merged to main? Because this bug seems to occur on gecko as well.

**Comment by mshin-wolvic** [comment]
> What do you think this PR be merged to main? Because this bug seems to occur on gecko as well.

We could merge it into the main branch after getting approval.

**Review by duswnchl — APPROVED** [review]

**Review by svillar — APPROVED** [review]
Awesome

**Pull request merged**
